### PR TITLE
Document too little memory for LUKS setup

### PR DESCRIPTION
**Pull request by VladimirSlavik**

### CODE REVIEW

File: docs/common-bugs.rst [hunk 1]
```diff
@@ -214,6 +214,24 @@ Undetected partitions
...
+
+          WARNING:blivet:Less than (...) MiB RAM is currently free, LUKS2 format may fail.
+          ui.gui.spokes.storage: Partitioning has been applied: ValidationReport(error_messages=[],
+            warning_messages=['The available memory is less than 128 MiB which can be too small
+            for LUKS2 format. It may fail.'])
+          Activating service name='org.fedoraproject.Anaconda.Modules.Storage'  (present more than once)
+
+    Note that the user must have ignored a warning in the GUI.
+:Workaround: There are several possible workarounds: Use more memory for the machine, use
+    ``--pbkdf*`` options in kickstart file, Change LUKS version to ``LUKS1``, disable encryption.
```
Comment (poncovka) [inline docs/common-bugs.rst:232]: `Change` -> `change`

Comment (poncovka) [reply docs/common-bugs.rst:232]: Maybe, it might be better to specify the items in a list.

**Review by poncovka — APPROVED** [review]
Otherwise, it looks good to me. Thanks!

File: docs/common-bugs.rst (hunk 1 above)
Comment (VladimirSlavik) [reply docs/common-bugs.rst]: Done...

**Review by poncovka — APPROVED** [review]
It looks great!

**Pull request merged**